src/amo/api/index.js

```diff
@@ -13,10 +50,16 @@ export function submitReview({
   body,
   reviewId,
   ...apiCallParams
-}) {
+}: SubmitReviewParams): Promise<ApiReviewType> {
   return new Promise(
     (resolve) => {
-      const data = { rating, version: versionId, body, title };
+      const data = {
```

Comment:
Nitpick but `postData`, `ratingData`, or `reviewData` might be a bit less generic. `data` is a vague variable name.